packages/console/src/pages/SignInExperience/tabs/Branding/CustomCssForm.module.scss

```diff
@@ -1,5 +1,5 @@
 .customCssCodeEditor {
   max-height: calc(100vh - 260px);
-  min-height: 111px; // min-height to show three lines of code
+  min-height: 132px; // min-height to show three lines of code
```

Comment:
Now min height fits a minimum of 4 LOC?

Reply:
i don't know how many lines actually. does it matter? if so, we'd better to use a stable value to specify row number instead of using magic number

Reply:
discuss offline